frontend/src/components/tables/DataTableHebrew.tsx

```
@@ -183,73 +221,88 @@ export function DataTableHebrew<TData, TValue>({
-                  אין תוצאות
-                </TableCell>
-              </TableRow>
-            )}
-          </TableBody>
-        </Table>
+            <TableBody>
+              {table.getRowModel().rows?.length ? (
+                table.getRowModel().rows.map((row) => (
+                  <UserExpiredTooltip isActive={handleHoverOnRow?.(row.original)}>
+                    <TableRow
+                      key={row.id}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)